site/app/templates/calendar/Calendar.twig

```diff
@@ -31,6 +32,59 @@
         }
     });
 
+    function changeYear(x) {
```

Comment:
These functions (and changeView below) should really be moved to a js file. We want <script> tags to be minimal, so if it can go in a js file it should.